packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -193,6 +200,10 @@ export class RecentsWidget extends Component {
               });
             });
         }
+
+        // Fetch notification features
+        props.getFeature(USER, GROUP_MESSAGE_NOTIFICATIONS, sparkInstance);
```

Comment:
This is going to fetch the features every time the props update, do we need to create a widget status for tracking?

Reply:
I'll have to dig around for a widget status that it may use to only fetch once during an init stage

Reply:
Take a look at the `isFetchingTeams` and `hasFetchedTeams` statuses. You'll want to use that same method but add a `hasFetchedNotificationsFeature`.